### PR TITLE
Alt bedpres

### PR DESCRIPTION
**Pull request by SmithPeder**
Resolves https://github.com/webkom/lego/issues/1628

Depends on https://github.com/webkom/lego/pull/1634

image

image

### CODE REVIEW

File: app/routes/events/utils.js [hunk 1]
```diff
@@ -1,36 +1,45 @@
...
-import type { TransformEvent, EventType } from 'app/models';
+import type { TransformEvent } from 'app/models';
 
-export const eventTypes = {
-  company_presentation: 'Bedriftspresentasjon',
+// Value type based on the EVENT_CONSTANTS
+export type eventTypes = $Values<typeof EVENT_CONSTANTS>;
+
+// Current eventTypes
+export const EVENT_CONSTANTS: eventTypes = {
```
Comment (odinuge) [inline app/routes/events/utils.js]: Smooth! Less duplication :ok_hand:

File: app/routes/events/utils.js [hunk 2]
```diff
@@ -1,36 +1,45 @@
...
+  company_presentation: 'Bedpres',
```
Comment (odinuge) [inline app/routes/events/utils.js]: Not sure, but think i prefer `Bedriftspresentasjon`, but for `Alternativ Bedriftspresentasjon` may be a bit too long. Have you tested how that works (and maybe increase the size of the list. :thinking:

**Comment by odinuge** [comment]
cc @regineru

File: app/routes/events/utils.js (hunk 2 above)
Comment (SmithPeder) [reply app/routes/events/utils.js]: I also prefer `Bedriftspresentasjon`, but as you say `Alternativ Bedriftspresentasjon` is quite long. I tested that and the text will wrap almost everywhere it's used.

Not sure what you mean by "_and maybe increase the size of the list_"? Like the number of elements or the width, to remove the wrapping?

Comment (odinuge) [reply app/routes/events/utils.js]: > and maybe increase the size of the list

0/10 phrasing 😅

> I tested that and the text will wrap almost everywhere it's used.


Yee, "Alternativ bedpres" is probably the way to go.

File: app/routes/events/utils.js [hunk 3]
```diff
@@ -1,36 +1,45 @@
...
-  other: 'Annet',
-  event: 'Arrangement'
+  event: 'Arrangement',
+  kid_event: 'KID-arrangement',
+  other: 'Annet'
 };
 
-export const EVENT_TYPE_TO_STRING = (eventType: EventType) => {
-  return eventTypes[eventType] || eventTypes['other'];
+// Returns the sting representation of an eventType
```
Comment (hanswilw) [inline app/routes/events/utils.js]: I hate when the eventType stings :trollface:

Comment (SmithPeder) [reply app/routes/events/utils.js]: 🙃 `:set spell spelllang=en_us` 🙃

File: app/routes/events/utils.js [hunk 4]
```diff
@@ -1,36 +1,45 @@
...
+import type { TransformEvent } from 'app/models';
 
-export const eventTypes = {
-  company_presentation: 'Bedriftspresentasjon',
+// Value type based on the EVENT_CONSTANTS
+export type eventTypes = $Values<typeof EVENT_CONSTANTS>;
+
+// Current eventTypes
+export const EVENT_CONSTANTS: eventTypes = {
+  company_presentation: 'Bedprespresentasjon',
```
Comment (odinuge) [inline app/routes/events/utils.js]: `Bedprespresentasjon` 🤔

**Comment by SmithPeder** [comment]
Ohh, holy smoke :smiley:

**Pull request merged**